src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -62,14 +63,14 @@ public static InstanceMappingFileSettings InstanceMappingFile(this RoutingSettin
         }
 
         /// <summary>
-        /// Overrides the default address translation rule "endpoint.quailfier-id@machine".
+        /// Registers the provided endpoint instances in the routing system.
         /// </summary>
         /// <param name="config">Config object.</param>
-        /// <param name="translationRule">New translation rule.</param>
-        public static TransportExtensions<MsmqTransport> OverrideAddressTranslation(this TransportExtensions<MsmqTransport> config, Func<LogicalAddress, string> translationRule)
+        /// <param name="instances">A list of endpoint instances.</param>
+        public static void AddEndpointInstances(this RoutingSettings<MsmqTransport> config, params EndpointInstance[] instances)
```

Comment:
we essentially wanted to avoid hardcoding instances on the basic API and now we introduce this again?